package.json

```
@@ -35,6 +35,7 @@
     "publish:studio": "yarn changeset publish --tag studio",
     "version:latest": "yarn changeset version && yarn generateVersions && yarn angular build",
     "publish:latest": "yarn changeset publish",
+    "version:hotfix": "yarn changeset && yarn bumpVersions && yarn build",
```

Don't think `yarn changeset` is needed unless I'm missing something:

```suggestion
    "version:hotfix": "yarn bumpVersions && yarn build",
```

---

Running changeset is needed bc otherwise `bumpVersions` doesn't have any changeset to consume. This script is meant to be run locally before pushing the hotfix PR.

---

Ah, gotcha, makes sense.